assets/report/report.qmd

```diff
@@ -504,3 +508,34 @@ For scores from the PGS Catalog, please remember to cite the original publicatio
 > PGS Catalog Calculator (in development). PGS Catalog Team. `https://github.com/PGScatalog/pgsc_calc`
 
 > Lambert et al. (2021) The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation. Nature Genetics. 53:420–425 doi:10.1038/s41588-021-00783-5.
+
+## Score licenses
```

Comment:
Why not put the licences into the first scores metadata table?

Reply:
the license text is a little bit big!